### ISSUE TITLE
Crosspost not running after node run

### ISSUE THREAD

**Issue by theuiguru** [humanwhocodes/crosspost#11]
I copied the following code and replaced all the credentials in my node index.js file:
```
import {
	Client,
	TwitterStrategy,
	BlueskyStrategy,
} from "@humanwhocodes/crosspost";

// Note: Use an app password, not your login password!
const bluesky = new BlueskyStrategy({
	identifier: "[myusername].bsky.social",
	password: "your-app-password",
	host: "bsky.social",
});

// Note: OAuth app is required
const twitter = new TwitterStrategy({
	accessTokenKey: "access-token-key",
	accessTokenSecret: "access-token-secret",
	apiConsumerKey: "api-consumer-key",
	apiConsumerSecret: "api-consumer-secret",
});

// create a client that will post to all three
const client = new Client({
	strategies: [bluesky, twitter],
});

// post to all two
await client.post("Hello world!");
```

I also have installed 'npm install @humanwhocodes/crosspost' in node project too. After I ran "node .", I got the following error with module not found:
<img width="528" alt="Screenshot 2024-11-21 at 3 46 42 PM" src="https://github.com/user-attachments/assets/0196facb-3224-4044-9496-8ea1b51fef4c">

AI is recommending to import a different way: 'const crosspost = require('@humanwhocodes/crosspost');' but not sure if that's a good approach.

**Comment by nzakas** [comment]
Weird. It looks like the files weren't include in `npm publish`. 🤔

**Comment by nzakas** [comment]
Fixed in 892ce016966d30829188530104782c7c15478a2b
